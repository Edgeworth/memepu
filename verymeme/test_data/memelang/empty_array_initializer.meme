fn main() i8 {
  var array: [5]i8 = [];
}
