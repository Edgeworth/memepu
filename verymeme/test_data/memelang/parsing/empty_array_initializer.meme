fn main() i8 {
  array : [5]i8 = {};
}
