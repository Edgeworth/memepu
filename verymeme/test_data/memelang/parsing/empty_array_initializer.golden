File
 Fn
  FnSig
   Typename(main)
   Type(i8)
  StmtBlk
   VarDefn
    VarDecl
     VarRef(array)
     Type(i8)
      Qualifier(const: 0 ptr: 0, array: IntLit(5))
       IntLit(5)
    CompoundLit
